scripts/installer/machine/x86_64-accton_csp7551-r0/platform.conf

```diff
@@ -0,0 +1,37 @@
+GRUB_CMDLINE_LINUX="console=tty0 console=ttyS0,115200n8"
+GRUB_SERIAL_COMMAND="serial --port=0x3f8 --speed=115200 --word=8 --parity=no --stop=1"
+EXTRA_CMDLINE_LINUX=""
+BISDN_LINUX_VOLUME_SIZE_MB=163840
+
+platform_setup() {
+    (cat <<EOF
+[Match]
+Path=pci-0000:18:00.0
+
+[Link]
+Description=Tofino ICE port (aka enp24s0)
```

Comment:
`enp24s0` is systemd naming scheme dependent name, and can change with newer/older systemd versions.

Please choose a better identifier here (maybe there is a label on the box?)

Likewise all the other ones.

Reply:
These are internal ports for communication between CPU and Tofino, so there are no labels. I thought the names given by systemd were still useful, but I have removed them now.